src/amoc_controller.erl

```diff
@@ -24,17 +26,21 @@
                 tref :: timer:tref() | undefined}).
 
 -type state() :: #state{}.
+%% Internal state of the node's controller
 -type handle_call_res() :: ok | {ok, term()} | {error, term()}.
 -type amoc_status() :: idle |
                        {running, amoc:scenario(), user_count(), last_user_id()} |
                        {terminating, amoc:scenario()} |
                        {finished, amoc:scenario()} |
                        {error, any()} |
-                       disabled. %% amoc_controller is disabled for the master node
-
+                       disabled.
+%% Status of the node, note that amoc_controller is disabled for the master node
 -type user_count() :: non_neg_integer().
+%% Number of users currently running in the node
 -type last_user_id() :: non_neg_integer().
+%% Highest user id registered in the node
 -type interarrival() :: non_neg_integer().
+%% Time to wait in between spawning new users
```

Comment:
These comments are bellow the types they describe. To keep it consistent with e.g. functions documentation I'd place them above instead.

Reply:
The funny thing, is that it needs to be below. ex_doc will draw the line as a summary under the type if it comes _after_ the type 🥲 
Try building the docs locally to see 🤷🏽

Reply:
Oh I see. Thank you for explaining this.